DetectorDescription/DDCMS/src/DDFilteredView.cc

```diff
@@ -7,14 +7,15 @@
 #include <TGeoBBox.h>
 #include <TGeoBoolNode.h>
 #include <vector>
+#include <charconv>
 
 using namespace cms;
 using namespace edm;
 using namespace std;
 using namespace cms::dd;
 
 dd4hep::Solid DDSolid::solidA() const {
-  if (dd4hep::isA<dd4hep::SubtractionSolid>(solid_) || dd4hep::isA<dd4hep::UnionSolid>(solid_) ||
+  if (dd4hep::isA<dd4hep::SubtractionSolid>(solid_) or dd4hep::isA<dd4hep::UnionSolid>(solid_) or
```

Comment:
Are you really going to be changing all the `||` to `or` in our code? I don't see the point. `||` is so well-established that `or` does not enhance clarity.

Reply:
I think, for a younger generation of the C++ programmers the former would look rather weird while the latter is just plain English :-)

Reply:
But a combination of the two is the worst of both worlds.

Reply:
> I think, for a younger generation of the C++ programmers the former would look rather weird while the latter is just plain English :-)

I'm feeling older after reading this. I'll miss the pipes :/